src/nvim/eval.c

```diff
@@ -15509,6 +15451,35 @@ static void f_sort(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   do_sort_uniq(argvars, rettv, true);
 }
 
+/// "stdioopen()" function
+static void f_stdioopen(typval_T *argvars, typval_T *rettv, FunPtr fptr)
```

Comment:
It's not that different, it's opening a channel. We don't need numerous different narrowly-defined functions to open a channel. A `chanopen` function can work like `jobstart`, by taking options that control the behavior.